staging/src/k8s.io/api/scheduling/v1alpha1/types.go

```diff
@@ -39,6 +39,9 @@ type PriorityClass struct {
 
 	// globalDefault specifies whether this PriorityClass should be considered as
 	// the default priority for pods that do not have any priority class.
+	// Only one PriorityClass with can be marked as `globalDefault`. Due to race conditions, more than
```

Comment:
Calling this a race condition makes it sound like a bug, which it isn't. I would just say that only one should have it, and if multiple do, this is the behavior

Reply:
Done. PTAL.